src/clr-core/styles/mixins/_mixins.scss

```diff
@@ -187,47 +187,58 @@ $opp-directions: (
 
 // line-height erasers
 
-@function getLineHeightGap($line-height-value: 1.2em, $line-height-token: null) {
+@function getLineHeightGap($line-height-value: 1.2, $line-height-token: null) {
   @if ($line-height-token == null) or ($line-height-value < 1) {
     @return 0;
   }
   @return calc((#{$line-height-token} - 1em) / 2);
 }
 
-@function pullUp($line-height-gap: 1.2em, $font-top-gap: 0.147em) {
-  @return calc((#{$font-top-gap} + #{$line-height-gap}) * -1);
+@function pullUp($line-height-gap: 1.2em, $font-top-gap: 0.1475em, $pull-nudge: 0.037em) {
+  @return calc(((#{$font-top-gap} + #{$line-height-gap}) * -1) + #{$pull-nudge});
 }
 
 @function pushDown(
   $line-height-gap: 0.6em,
-  $font-top-gap: 0.147em,
-  $font-ascender-height: 0.17,
-  $font-x-height: 0.517
+  $font-top-gap: 0.1475em,
+  $font-ascender-height: 0.1703,
+  $font-x-height: 0.517,
+  $push-nudge: 0.044em
 ) {
-  @return calc(((1em - #{$font-top-gap} - #{$font-ascender-height} - #{$font-x-height}) + #{$line-height-gap}) * -1);
+  @return calc(
+    (((1em - #{$font-top-gap} - #{$font-ascender-height} - #{$font-x-height}) + #{$line-height-gap}) * -1) - #{$push-nudge}
+  );
 }
 
 // default are for metropolis/clarity-city
 @mixin line-height-eraser(
```

Comment:
So were we not able to use the line height eraser for the button or was it not needed? Do we need to have an inner span for projected content to use the line height eraser?

Reply:
LHEs were behaving erratically. I can try to find a workaround for that but badges weren't behaving much better. The difference appears to be precise vertical sizing versus flow. LHEs do flow but disrupt precise vertical sizing.